.circleci/config.yml

```diff
@@ -52,44 +32,27 @@ jobs:
           root: ~/repo
           <<: *whitelist
 
-  build_base_docker:
-    <<: *defaults
-
-    steps:
-      - attach_workspace:
-          at: ~/repo
-
-      - setup_remote_docker
-
-      - run:
-          name: Build Base Docker
-          # install dependencies for loading ecs task definitions
-          command: |
-            eval $(aws ecr get-login --no-include-email)
-            globality-build build-gen local
-            globality-build docker --repo python-library
-
   test:
-    <<: *defaults
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:3.7.0
+      - image: circleci/postgres:9.6.5-alpine-ram
```

Comment:
CircleCI has a lot of magic out of the box. Specifying this image and the environment variables below does everything we need for the tests (which use postgres) to work. See: https://circleci.com/docs/2.0/databases/ In general if you need to do anything unusual or understand a stanza in the `circle.yml` config that isn't obvious, it's worth looking at the circle docs as there is usually a good explanation for the API that is being used.